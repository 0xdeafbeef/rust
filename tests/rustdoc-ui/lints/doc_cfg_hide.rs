#![doc(auto_cfg(hide = "test"))] //~ ERROR
#![doc(auto_cfg(hide))] //~ ERROR
