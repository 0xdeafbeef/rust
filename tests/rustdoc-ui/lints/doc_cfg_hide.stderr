error: `#![doc(auto_cfg(hide(...)))]` only expects a list of items
  --> $DIR/doc_cfg_hide.rs:1:8
   |
LL | #![doc(auto_cfg(hide = "test"))]
   |        ^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[deny(invalid_doc_attributes)]` on by default

error: `#![doc(auto_cfg(hide(...)))]` only expects a list of items
  --> $DIR/doc_cfg_hide.rs:2:8
   |
LL | #![doc(auto_cfg(hide))]
   |        ^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

